src/main/java/com/opencart/driver/DriverRepository.java

```diff
@@ -20,11 +21,11 @@ private DriverRepository() {
     }
 
     public static void downloadWebDriver() {
-        WebDriverManager.chromedriver().setup();
+        WebDriverManager.firefoxdriver().setup();
```

Comment:
If you want to use Firefox, you can create a switch case, and take the browser name as a parameter of downloadWebDriver() method.